src/test/java/org/example/c01operators/calculate/DaysToYearsWeeksAndDaysTest.java

```diff
@@ -5,29 +5,31 @@
 import static org.junit.jupiter.api.Assertions.*;
 
  /*
+ Write a code in Java to transform numbers of days in years, weeks and days.
+
  I create an arrays with length=3. First position is for number of years
         second for number of weeks and last one for number of days
 */
 
 public class DaysToYearsWeeksAndDaysTest {
     @Test
-    public void tenLeapYear() {
+    public void given3660AsNumberOfDaysShouldBeTenLeapYear() {
         int numberOfDays = 3660;
         int[] resultYearsWeeksDays = {10, 0, 0};
         int[] result = DaysToYearsWeeksAndDays.findYearsWeeksDays(numberOfDays);
         assertArrayEquals(result, resultYearsWeeksDays);
     }
 
     @Test
-    public void nineHundredDays() {
+    public void given900AsNumberOfDaysShouldBe2Years24WeeksAnd2Days() {
```

Comment:
900 is to complex input. Try with input 1 or7 or 8 or 365 or 373 or 1461(4 years of witch one is a leap year)

Reply:
done

Reply:
where?